setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = peek_data
+author = Scipp contributors (https://github.com/scipp)
+description =
+license = BSD
+license_files = LICENSE
+long_description = file: README.md
+long_description_content_type = text/markdown
+url =
+project_urls =
+    Bug Tracker = https://github.com/scipp/peek-data/issues
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+install_requires =
+python_requires = >=3.8
+include_package_data = True
+
+[options.packages.find]
+where = src
+
+[flake8]
+# See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
+max-line-length = 88
+extend-ignore = E203
+exclude = .*,__pycache__,dist
```

Comment:
Please change to the more modern version and move everything you can into `pyproject.toml`. See ScippNeutron.

Reply:
Though ScippNeutron might also be out of date. Setuptools is in the process of migrating everything to pyproject.toml.